integration-test/1185-garden-min-zoom.py

```diff
@@ -0,0 +1,10 @@
+# this garden previously had a min_zoom of 12, but based on its size should be
+# z16 instead.
+#
+# http://www.openstreetmap.org/way/273274870
+assert_has_feature(
+    16, 32182, 20422, 'landuse',
+    { 'kind': 'garden', 'id': 273274870, 'min_zoom': 16, 'tier': 6 })
+assert_has_feature(
```

Comment:
This garden is marked `access=private` and doesn't have a name so I was only expecting to see a polygon in the `landuse` layer and no feature in the `pois` layer.